### ISSUE TITLE
Perpetual issue for old/dead links

### ISSUE THREAD

**Issue by gtjoseph** [asterisk/documentation#59]
If you find a link to the old wiki.asterisk.org website that just takes you to the new documentation site homepage instead of real content, add it here and we'll try to set up a redirect that takes you to the relevant content instead.

Please include the stale link and, if you found the real content on the new website, that link as well.

**Comment by oduserdnov** [comment]
https://docs.asterisk.org/Configuration/Dialplan/Special-Dialplan-Extensions/ page has `EXCEPTION` link with URL https://docs.asterisk.org/latest_api/API_Documentation/Dialplan_Functions/EXCEPTION but I bet it must be https://docs.asterisk.org/Latest_API/API_Documentation/Dialplan_Functions/EXCEPTION (manually detected).

`latest_api` -> `Latest_API`. Maybe worth to find others like that (just in case).

**Comment by 2br-2b** [comment]
https://wiki.asterisk.org/wiki/display/AST/Migrating+from+chan_sip+to+res_pjsip (shouldn't be needed after Asterisk v21, but still good to have)

I couldn't find this content on the new site

**Comment by jcolp** [comment]
It's here: https://docs.asterisk.org/Configuration/Channel-Drivers/SIP/Configuring-res_pjsip/Migrating-from-chan_sip-to-res_pjsip/

**Comment by Starvingg** [comment]
Hello, I couldn't find the documentation for Dialplan SendFax and Dialplan RecieveFax

https://docs.asterisk.org/Asterisk_16_Documentation/API_Documentation/Dialplan_Applications/SendFax
https://docs.asterisk.org/Asterisk_16_Documentation/API_Documentation/Dialplan_Applications/ReceiveFax

**Comment by oduserdnov** [comment]
> Hello, I couldn't find the documentation for Dialplan SendFax and Dialplan RecieveFax
> 
> https://docs.asterisk.org/Asterisk_16_Documentation/API_Documentation/Dialplan_Applications/SendFax https://docs.asterisk.org/Asterisk_16_Documentation/API_Documentation/Dialplan_Applications/ReceiveFax

Expected links:
https://docs.asterisk.org/Asterisk_16_Documentation/API_Documentation/Dialplan_Applications/SendFAX
https://docs.asterisk.org/Asterisk_16_Documentation/API_Documentation/Dialplan_Applications/ReceiveFAX

Looks like this is a common issue across the docs (case does not match).

Btw, I bet it would be better to mention where is the starting point for your links @Starvingg , as I had no issues to locate working ones but yours - have no idea where did you come from.

**Comment by Starvingg** [comment]
Hey @oduserdnov below is my starting point (At the end of the document)
https://docs.asterisk.org/Asterisk_16_Documentation/API_Documentation/Dialplan_Functions/FAXOPT/

I tried searching as well but wasn't confident the _ReceiveFax_ and _SendFax_ I found was related to Dialplan.

**Comment by gtjoseph** [comment]
Ah.  Those need to be fixed in the res_fax module itself.  I'll get a fix pushed up.

**Comment by gtjoseph** [comment]
The case mismatches for dialplan applications and functions should be fixed in all branches now.  I had forgotten to do the 16 branch the other day but it's fixed there now as well.

**Comment by remiquezada** [comment]
https://wiki.asterisk.org/wiki/display/DIGIUM/Digium+D-Series+Phones

**Comment by jcolp** [comment]
That was moved to https://sangomakb.atlassian.net/wiki/spaces/Phones/pages/21299305/Sangoma+D-Series+Phones as it is Sangoma commercial product documentation, and not part of Asterisk or open source.

**Comment by 2br-2b** [comment]
https://wiki.asterisk.org/wiki/display/AST/Configuring+res_pjsip (along with the sip link I mentioned earlier) are both referenced in the deprecation notice printed by Asterisk

**Comment by linuxfingers** [comment]
Hey all! Found a redirect issue for the Hangup Handlers documentation.

Link I was looking for: 
https://wiki.asterisk.org/wiki/display/AST/Hangup+Handlers

Link that it was redirected to: 
https://docs.asterisk.org/Hangup+Handlers

Actual location of information:
https://docs.asterisk.org/Configuration/Dialplan/Subroutines/Hangup-Handlers/

**Comment by gtjoseph** [comment]
> https://wiki.asterisk.org/wiki/display/AST/Configuring+res_pjsip (along with the sip link I [mentioned earlier](https://github.com/asterisk/documentation/issues/59#issuecomment-2023430388)) are both referenced in the deprecation notice printed by Asterisk

@2br-2b 
I thought we squashed all of those.  Which branch are you seeing the message in and for which module?

**Comment by gtjoseph** [comment]
> Hey all! Found a redirect issue for the Hangup Handlers documentation.
> 
> Link I was looking for: https://wiki.asterisk.org/wiki/display/AST/Hangup+Handlers
> 
> Link that it was redirected to: https://docs.asterisk.org/Hangup+Handlers
> 
> Actual location of information: https://docs.asterisk.org/Configuration/Dialplan/Subroutines/Hangup-Handlers/

Thanks.  I'm trying to find a better way to handle the redirects.

**Comment by DrPyser** [comment]
https://docs.asterisk.org/Asterisk_certified/18.9_Documentation/API_Documentation/Dialplan_Applications/WaitExten

Redirected to this page from the `See Also` section of [Playback](https://docs.asterisk.org/Certified-Asterisk_18.9_Documentation/API_Documentation/Dialplan_Applications/Playback/) for the `Dialplan Applications WaitExten` link.

Seems the path prefix `Asterisk_certified/18.9_Documentation` should be `Certified-Asterisk_18.9_Documentation`: https://docs.asterisk.org/Certified-Asterisk_18.9_Documentation/API_Documentation/Dialplan_Applications/WaitExten/

**Comment by gtjoseph** [comment]
Thanks for the report @DrPyser .  These should all be fixed now.

**Comment by ronlockard** [comment]
https://wiki.asterisk.org/wiki/display/AST/Hangup+Cause+Mappings

should redirect to

https://docs.asterisk.org/Configuration/Miscellaneous/Hangup-Cause-Mappings/?h=hangup+cause

**Comment by gtjoseph** [comment]
Thanks @ronlockard.  I'm trying to find a better way to handle the redirects that don't fit a known pattern.

**Comment by bguilera** [comment]
https://wiki.asterisk.org/wiki/display/AST/SIP+Retransmissions

Currently Redirects to which has no info:

https://docs.asterisk.org/SIP+Retransmissions

Probably should redirect to:

https://docs.asterisk.org/Development/Roadmap/Asterisk-16-Projects/WebRTC-User-Experience-Improvements/?h=retransmission#rtp-packet-retransmission

**Comment by gtjoseph** [comment]
@bguilera Had you visited that page on the old wiki at some time?  I don't see it in the dump we took so I'm wondering what it pointed to originally.

**Comment by bguilera** [comment]
@gtjoseph Unfortunately no, I don't recall visiting the old wiki.

**Comment by sailor-ITlab** [comment]
On this Site: [https://docs.asterisk.org/Getting-Started/Installing-Asterisk/Installing-Asterisk-From-Source/What-to-Download/](url)
is an Link "Checking Asterix Requirements" which redirects to this non existent site: [https://docs.asterisk.org/Getting-Started/Installing-Asterisk/Installing-Asterisk-From-Source/Checking-Asterisk-Requirements](url)

**Comment by sailor-ITlab** [comment]
On this Site: [https://docs.asterisk.org/Getting-Started/Installing-Asterisk/Installing-Asterisk-From-Source/What-to-Download/](url)
is an Link at the download section, which then gives an Error 500: [https://downloads.asterisk.org/pub/telephony/libpri/libpri-current.tar.gz](url)

**Comment by davidbqzt** [comment]
https://docs.asterisk.org/Asterisk+12+CDR+Specification#Asterisk12CDRSpecification-Fields
Thanks for your help

**Comment by paul-jewell** [comment]
On this page: https://docs.asterisk.org/Getting-Started/Hello-World/ the "installing pjproject" link is [Installing pjproject](https://docs.asterisk.org/Getting-Started/Installing-Asterisk/Installing-Asterisk-From-Source/PJSIP-pjproject), but I think it should be https://docs.asterisk.org/Getting-Started/Installing-Asterisk/Installing-Asterisk-From-Source/Prerequisites/PJSIP-pjproject/
Currently it shows the 404 page, linking to this issue.
Many thanks!

**Comment by supercooltest** [comment]
All the links at [https://www.asterisk.org/downloads/source-code/](https://www.asterisk.org/downloads/source-code/)
* [Gerrit repository](https://wiki.asterisk.org/wiki/display/AST/Gerrit+Usage)
* [installation guide on the Wiki](https://wiki.asterisk.org/wiki/display/AST/Installing+Asterisk+From+Source)
* [Gerrit Usage instructions](https://wiki.asterisk.org/wiki/display/AST/Gerrit+Usage)

**Comment by gtjoseph** [comment]
> All the links at https://www.asterisk.org/downloads/source-code/
> 
> * [Gerrit repository](https://wiki.asterisk.org/wiki/display/AST/Gerrit+Usage)
> * [installation guide on the Wiki](https://wiki.asterisk.org/wiki/display/AST/Installing+Asterisk+From+Source)
> * [Gerrit Usage instructions](https://wiki.asterisk.org/wiki/display/AST/Gerrit+Usage)

@supercooltest Thanks for reporting this.  This page isn't actually linked from www.asterisk.org any more which is probably why we never updated it.  We didn't know it was there.  I take it you found this page linked from an external site?  Anyway, I've updated it.

**Comment by gtjoseph** [comment]
> Old link: https://docs.asterisk.org/Asterisk+Community+Code+of+Conduct New link: https://docs.asterisk.org/Asterisk-Community/Asterisk-Community-Code-of-Conduct/

Thanks, I'll try and get that fixed on Monday when I'm back in the office.

**Comment by hanyuanzhong** [comment]
Hello, no link found: https://wiki.asterisk.org/wiki/display/AST/External+Media+and+ARI 

The link from https://community.asterisk.org/t/streaming-from-ari-snoop-channel-for-speech-recognition/75982/6

**Comment by gtjoseph** [comment]
> Hello, no link found: https://wiki.asterisk.org/wiki/display/AST/External+Media+and+ARI
> 
> The link from https://community.asterisk.org/t/streaming-from-ari-snoop-channel-for-speech-recognition/75982/6

I fixed the community post to point to the correct page.

**Comment by gtjoseph** [comment]
> Old link: https://docs.asterisk.org/Asterisk+Community+Code+of+Conduct New link: https://docs.asterisk.org/Asterisk-Community/Asterisk-Community-Code-of-Conduct/

Redirect added.  Should show up shortly.

**Comment by Rayan-Weragala** [comment]
Old link found : https://docs.asterisk.org/Function_TIMEOUT

**Comment by DrPyser** [comment]
https://docs.asterisk.org/Certified-Asterisk_18.9_Documentation/API_Documentation/Dialplan_Applications/Background

Linked from https://docs.asterisk.org/Certified-Asterisk_18.9_Documentation/API_Documentation/Dialplan_Applications/Playback/

**Comment by gtjoseph** [comment]
> Old link found : https://docs.asterisk.org/Function_TIMEOUT
@Rayan-Weragala Where did you find that link?

**Comment by gtjoseph** [comment]
> https://docs.asterisk.org/Certified-Asterisk_18.9_Documentation/API_Documentation/Dialplan_Applications/Background
> 
> Linked from https://docs.asterisk.org/Certified-Asterisk_18.9_Documentation/API_Documentation/Dialplan_Applications/Playback/

This needed to be fixed in Asterisk itself.  Should propagate here in a few days.

**Comment by chorons6bo** [comment]
Hello,
Did not find any more :
https://wiki.asterisk.org/wiki/display/AST/Building+and+Installing+Asterisk

**Comment by gtjoseph** [comment]
@chorons6bo You're correct.  I've added a redirect for the original page and it should become active sometime today.

**Comment by wonder75** [comment]
Not working:
https://docs.asterisk.org/Secure+Calling+Tutorial

**Comment by gtjoseph** [comment]
> Not working:
> https://docs.asterisk.org/Secure+Calling+Tutorial
> It might have moved here?
> https://docs.asterisk.org/Deployment/Secure-Calling/Secure-Calling-Tutorial/

Yep.  I just added a redirect for it.  It should be active in a few minutes.

**Comment by suretelZA** [comment]
Good day,
We are looking for this link which is often referred to for diversion headers in asterisk.
https://wiki.asterisk.org/wiki/display/AST/Manipulating+Party+ID+Information#ManipulatingPartyIDInformation-REDIRECTINGdialplanfunction

**Comment by jcolp** [comment]
https://docs.asterisk.org/Configuration/Functions/Manipulating-Party-ID-Information/?h=redirecting

**Comment by John-H-Smith** [comment]
https://docs.asterisk.org/PJSIP+Configuration+Wizard
should be redirected to 
https://docs.asterisk.org/Configuration/Channel-Drivers/SIP/Configuring-res_pjsip/PJSIP-Configuration-Wizard/?h=wizard

**Comment by gtjoseph** [comment]
> https://docs.asterisk.org/PJSIP+Configuration+Wizard should be redirected to https://docs.asterisk.org/Configuration/Channel-Drivers/SIP/Configuring-res_pjsip/PJSIP-Configuration-Wizard/?h=wizard

@John-H-Smith Thanks for the report!  Redirect added.  Should be live in a few minutes.

**Comment by sn4kebite** [comment]
https://wiki.asterisk.org/wiki/display/AST/ConfBridge
redirects to
https://docs.asterisk.org/ConfBridge
which gives 404. I found the actual documentation on
https://docs.asterisk.org/Configuration/Applications/Conferencing-Applications/ConfBridge/

**Comment by wphowell** [comment]
https://docs.asterisk.org/Realtime+Database+Configuration#RealtimeDatabaseConfiguration-Limitations

**Comment by gtjoseph** [comment]
> https://wiki.asterisk.org/wiki/display/AST/ConfBridge redirects to https://docs.asterisk.org/ConfBridge which gives 404. I found the actual documentation on https://docs.asterisk.org/Configuration/Applications/Conferencing-Applications/ConfBridge/

@sn4kebite Thanks for the report.  Fixed.

> https://docs.asterisk.org/Realtime+Database+Configuration#RealtimeDatabaseConfiguration-Limitations

@wphowell Thanks for the report.  Kinda fixed. :)  You now get redirected to the correct page but the anchor to the "Limitations" section isn't working.  This is a limitation of the mkdocs redirect capability.  We may be able to fix this in the future.

**Comment by thicsilva** [comment]
https://docs.asterisk.org/Asterisk_20_Documentation/API_Documentation/AMI_Actions/SIPpeers/ gives 404. In previous versions the command is not listed in the documentation... Has it been removed?

**Comment by gtjoseph** [comment]
@thicsilva Fixed now.  Good catch, thanks.

**Comment by telsberry** [comment]
#asterisk IRC code of conduct - bit.ly/1hH6P22
resolves to:
https://docs.asterisk.org/Asterisk+Community+Code+of+Conduct
but content is at:
https://docs.asterisk.org/Asterisk-Community/Asterisk-Community-Code-of-Conduct/

**Comment by gtjoseph** [comment]
@telsberry Thanks for the report.  I _had_ a redirect for that already but it had a typo in it.  It should be fixed when the publish process runs tonight.

**Comment by paradoxheart** [comment]
https://docs.asterisk.org/Getting-Started/Beginning-Asterisk/ instructs users to visit the Wiki for up-to-date information, but the links are to `//`

**Comment by gtjoseph** [comment]
@paradoxheart Thanks for the report.  I've corrected the wording and links.  It should be fixed when the publish process runs tonight.

**Comment by gtjoseph** [comment]
@samhilmw Thanks for the report.  I've fixed the redirect and it should be live when the publish process runs tonight.

**Comment by rodaimi** [comment]
https://wiki.asterisk.org/wiki/display/AST/Important+Security+Considerations -> https://docs.asterisk.org/Deployment/Important-Security-Considerations/

**Comment by renato-mm** [comment]
https://wiki.asterisk.org/wiki/display/AST/Asterisk+11+AMI+Events

https://wiki.asterisk.org/wiki/display/AST/Asterisk+11+AMI+Actions

**Comment by basha0110** [comment]
https://wiki.asterisk.org/wiki/display/AST/SIP+Retransmissions
